xrpl/models/requests/request.py

```diff
@@ -9,14 +9,16 @@
 from enum import Enum
 from typing import Any, Dict, Optional, Type, Union, cast
 
-from typing_extensions import Self
+from typing_extensions import Final, Self
 
 import xrpl.models.requests  # bare import to get around circular dependency
 from xrpl.models.base_model import BaseModel
 from xrpl.models.exceptions import XRPLModelException
 from xrpl.models.required import REQUIRED
 from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
 
+DEFAULT_API_VERSION: Final[int] = 2
```

Comment:
```suggestion
_DEFAULT_API_VERSION: Final[int] = 2
```
It's a helper variable that's only used here (other than in tests).

Reply:
fixed in 4179077